packages/itwinui-react/src/core/Tile/Tile.tsx

```diff
@@ -137,148 +138,165 @@ export type TileProps = {
-        </div>
+            <span className='iui-tile-name-label'>{name}</span>
+          </div>
 
-        {description != undefined && (
-          <div className='iui-tile-description'>{description}</div>
-        )}
+          {description != undefined && (
+            <div className='iui-tile-description'>{description}</div>
+          )}
 
-        {metadata != undefined && (
-          <div className='iui-tile-metadata'>{metadata}</div>
-        )}
+          {metadata != undefined && (
+            <div className='iui-tile-metadata'>{metadata}</div>
+          )}
 
-        {moreOptions && (
-          <DropdownMenu
-            onShow={showMenu}
-            onHide={hideMenu}
-            menuItems={(close) =>
-              moreOptions.map((option: React.ReactElement) =>
-                React.cloneElement(option, {
-                  onClick: (value: unknown) => {
-                    close();
-                    option.props.onClick?.(value);
-                  },
-                }),
-              )
-            }
-          >
-            <div
-              className={cx('iui-tile-more-options', {
-                'iui-visible': isMenuVisible,
-              })}
+          {moreOptions && (
+            <DropdownMenu
+              onShow={showMenu}
+              onHide={hideMenu}
+              menuItems={(close) =>
+                moreOptions.map((option: React.ReactElement) =>
+                  React.cloneElement(option, {
+                    onClick: (value: unknown) => {
+                      close();
+                      option.props.onClick?.(value);
+                    },
+                  }),
+                )
+              }
             >
-              <IconButton
-                styleType='borderless'
-                size='small'
-                aria-label='More options'
+              <div
+                className={cx('iui-tile-more-options', {
+                  'iui-visible': isMenuVisible,
+                })}
               >
-                <SvgMore />
-              </IconButton>
-            </div>
-          </DropdownMenu>
-        )}
+                <IconButton
+                  styleType='borderless'
+                  size='small'
+                  aria-label='More options'
+                >
+                  <SvgMore />
+                </IconButton>
+              </div>
+            </DropdownMenu>
+          )}
+
+          {children}
+        </div>
 
-        {children}
+        {buttons && <div className='iui-tile-buttons'>{buttons}</div>}
       </div>
+    );
+  },
+  {
+    /**
+     * Polymorphic action component for actionable Tiles. Should be used in name.
+     * It is `button` by default. Can also be rendered as anchor link.
+     *
+     * @example
+     * <Tile
+     *   isActionable={isActionable}
+     *   name={<Tile.Action onClick={() => doOnClick()}>Tile name</Tile.Action>}
+     * />
+     */
+    Action: PolymorphicAction,
```

Comment:
does it need to be a subcomponent now? i think users could just pass the generic component directly